cmake/WaveUtils.cmake

```diff
@@ -1,6 +1,7 @@
 # - Macros used in libwave
 
 INCLUDE(CMakeParseArguments)
```

Comment:
Super nitpicky - but should this file be called WaveUtils.cmake? Could it be named WaveCMakeUtils.cmake? 

Just because we have a wave_utils package. Idk, just personal preference

Reply:
Good point, will rename WaveHelpers or something. Maybe in subsequent PR or history will be messed up.